tsdb/head.go

```diff
@@ -231,6 +235,26 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 	return m
 }
 
+const cardinalityCacheExpirationTime = time.Duration(30) * time.Second
+
+// Returns top 10 Highest Cardinality stats By label and value names.
```

Comment:
Comment format should be like `// PostingsCardinalityStats returns the top 10 highest cardinality stats by label and value names.`